### DISCUSSION TITLE
Additional Column - Current quantity

### DISCUSSION THREAD

**Discussion by Gizmo19699** [rgriebl/brickstore#716] [General]
Hi,

Would it be possible to add a column that would pull your current quantity of a part in your store?

Would help when doing an inventory check when putting away parts.

Cheers!
Mark

**Comment by ZZJHONS** [comment]
I tried to do this manually but was many steps:
- Make file to upload
- Copy that file contents to a new file
- Import your store
- Copy values to the file you are uploading
- Reset differences
- Copy values from the new file copied of upload content to the upload content file
- Now the difference values show the current inventory and the qty column shows what you are uploading.

But this is many steps and I stopped using it.

It would be great to have that information easier indeed.

+1
